--- umac.c	2020-09-27 09:25:01.000000000 +0200
+++ umac.c	2020-12-22 12:17:04.009729170 +0100
@@ -104,11 +104,13 @@
 /* For this implementation to behave correctly, __LITTLE_ENDIAN__ must    */
 /* be set true if the host computer is little-endian.                     */
 
+#if !defined(__LITTLE_ENDIAN__)
 #if BYTE_ORDER == LITTLE_ENDIAN
 #define __LITTLE_ENDIAN__ 1
 #else
 #define __LITTLE_ENDIAN__ 0
 #endif
+#endif
 
 /* ---------------------------------------------------------------------- */
 /* ---------------------------------------------------------------------- */
